.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,358 @@
+name: Publish Release
+
+on:
+    workflow_dispatch:
+        inputs:
+            environment:
+                type: environment
+                default: "Beta"
+            version:
+                type: string
+                description: "Version (semver) for the release."
+                default: "0.0.1"
+                required: true
```

Comment:
Do we still need this input, now that we're updating the version manually?

Reply:
You're right, we shouldn't need this.